pkg/services/ngalert/api/api.go

```diff
@@ -60,26 +61,27 @@ type AlertingStore interface {
 
 // API handlers.
 type API struct {
-	Cfg                  *setting.Cfg
-	DatasourceCache      datasources.CacheService
-	RouteRegister        routing.RouteRegister
-	ExpressionService    *expr.Service
-	QuotaService         *quota.QuotaService
-	Schedule             schedule.ScheduleService
-	TransactionManager   provisioning.TransactionManager
-	ProvenanceStore      provisioning.ProvisioningStore
-	RuleStore            store.RuleStore
-	InstanceStore        store.InstanceStore
-	AlertingStore        AlertingStore
-	AdminConfigStore     store.AdminConfigurationStore
-	DataProxy            *datasourceproxy.DataSourceProxyService
-	MultiOrgAlertmanager *notifier.MultiOrgAlertmanager
-	StateManager         *state.Manager
-	SecretsService       secrets.Service
-	AccessControl        accesscontrol.AccessControl
-	Policies             *provisioning.NotificationPolicyService
-	ContactPointService  *provisioning.ContactPointService
-	Templates            *provisioning.TemplateService
+	Cfg                    *setting.Cfg
+	DatasourceCache        datasources.CacheService
+	RouteRegister          routing.RouteRegister
+	ExpressionService      *expr.Service
+	QuotaService           *quota.QuotaService
+	Schedule               schedule.ScheduleService
+	TransactionManager     provisioning.TransactionManager
+	ProvenanceStore        provisioning.ProvisioningStore
+	RuleStore              store.RuleStore
+	InstanceStore          store.InstanceStore
+	AlertingStore          AlertingStore
+	AdminConfigStore       store.AdminConfigurationStore
+	DataProxy              *datasourceproxy.DataSourceProxyService
+	MultiOrgAlertmanager   *notifier.MultiOrgAlertmanager
+	StateManager           *state.Manager
+	SecretsService         secrets.Service
+	AccessControl          accesscontrol.AccessControl
+	Policies               *provisioning.NotificationPolicyService
+	ContactPointService    *provisioning.ContactPointService
+	Templates              *provisioning.TemplateService
+	NotificationDispatcher *sender.Dispatcher
```

Comment:
The alertmanager model calls these "Alerts" or "Alert Instances" - maybe `AlertDispatcher`? I think it would be good to make the meaning of "Notification" be "Something sent from Grafana to a Contact Point"

Reply:
Good Idea